astropy/table/tests/test_pickle.py

```diff
@@ -72,11 +72,11 @@ def test_pickle_table(protocol):
         assert np.all(tp['d'] == t['d'])
         assert np.all(tp['e'].ra == t['e'].ra)
         assert np.all(tp['e'].dec == t['e'].dec)
-        assert type(tp['c']) == type(t['c'])
-        assert type(tp['d']) == type(t['d'])
-        assert type(tp['e']) == type(t['e'])
+        assert type(tp['c']) is type(t['c'])  # nopep8
```

Comment:
not sure, it gives the same E721 warning as with ``==`` (``E721 do not compare types, use 'isinstance()'``), but it doesn't complain for simpler expression e.g. ``type(tp) is type(t)``